contact.html

```diff
@@ -5,13 +5,13 @@
         <title>YouClap - Challenge The World</title>
         <!-- Stylesheets -->
         <link href="//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet">
-        <link href="css/style.css" rel="stylesheet">
+        <link href="css/style.min.css" rel="stylesheet">
         <link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon">
         <link rel="icon" href="images/favicon.ico" type="image/x-icon">
         <!-- Responsive -->
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
         <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0">
-        <link href="css/responsive.css" rel="stylesheet">
+        <link href="css/responsive.min.css" rel="stylesheet">
```

Comment:
💄  can we merge the css links? 😇  i mean, move this line to after the `<link href="css/style.min.css" rel="stylesheet">`